### PR TITLE
update copytext for unsuccessful email notification

### PR DESCRIPTION
**Pull request by macool**

### CODE REVIEW

**Comment by macool** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;rmNDkLs6&#x2F;660-qae-support-email-to-unsuccessful-business-categories-registered-account-holder"> QAE Support: Email to UNSUCCESSFUL Business Categories Registered Account Holder</a>

File: app/views/users/unsuccessful_feedback_mailer/notify.html.slim [hunk 1]
```diff
@@ -1,20 +1,53 @@
+h4 style="font-size: 19px; font-weight:bold; line-height: 1.315789474; margin: 30px 0 30px 0;"
+  ' The Queen's Awards for Enterprise 2016
```
Comment (matthewford) [inline app/views/users/unsuccessful_feedback_mailer/notify.html.slim]: Can we make this the award year? e.g. not hard coded 2016

File: app/views/users/unsuccessful_feedback_mailer/notify.text.slim [hunk 2]
```diff
@@ -1,13 +1,39 @@
+' The Queen's Awards for Enterprise 2016
```
Comment (matthewford) [inline app/views/users/unsuccessful_feedback_mailer/notify.text.slim]: Ditto

File: app/views/users/unsuccessful_feedback_mailer/notify.text.slim [hunk 3]
```diff
@@ -1,13 +1,39 @@
...
+| [
+= @form_answer.urn
+| ]
+
 = "Dear #{@user.full_name},"
 
-= "Firstly, we'd like to thank you, and #{@company_name}, for taking the time to apply for The Queen's Awards for Enterprise: #{@form_answer.award_type_full_name}."
+' Thank you for your entry for The Queen's Awards for Enterprise 2016.
```
Comment (matthewford) [inline app/views/users/unsuccessful_feedback_mailer/notify.text.slim]: Ditto

File: app/views/users/unsuccessful_feedback_mailer/notify.html.slim [hunk 4]
```diff
@@ -1,20 +1,53 @@
...
+  = @form_answer.urn
+  | ]
+
 p style="font-size: 19px; line-height: 1.315789474; margin: 30px 0 30px 0;"
   = "Dear #{@user.full_name},"
 
-p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;"
-  = "Firstly, we'd like to thank you, and #{@company_name}, for taking the time to apply for The Queen's Awards for Enterprise: #{@form_answer.award_type_full_name}."
+p style="font-size: 19px; line-height: 1.315789474; margin: 30px 0 30px 0;"
+  ' Thank you for your entry for The Queen's Awards for Enterprise 2016.
```
Comment (matthewford) [inline app/views/users/unsuccessful_feedback_mailer/notify.html.slim]: Ditto

File: app/mailers/users/unsuccessful_feedback_mailer.rb [hunk 5]
```diff
@@ -2,9 +2,9 @@ class Users::UnsuccessfulFeedbackMailer < AccountMailer
   def notify(form_answer_id)
     @form_answer = FormAnswer.find(form_answer_id).decorate
     @user = @form_answer.user.decorate
```
Comment (rusllonrails) [inline app/mailers/users/unsuccessful_feedback_mailer.rb]: @macool note just for future:
Account holder is:
```
@user = @form_answer.account.owner.decorate
```

File: app/views/users/unsuccessful_feedback_mailer/notify.html.slim [hunk 6]
```diff
@@ -1,20 +1,56 @@
+h4 style="font-size: 19px; font-weight:bold; line-height: 1.315789474; margin: 30px 0 30px 0;"
+  ' The Queen's Awards for Enterprise
+  => @year
+  | [
+  = @form_answer.urn
+  | ]
+
 p style="font-size: 19px; line-height: 1.315789474; margin: 30px 0 30px 0;"
   = "Dear #{@user.full_name},"
```
Comment (rusllonrails) [inline app/views/users/unsuccessful_feedback_mailer/notify.html.slim]: @macool small note:
In attached Email_to_Business_Unsuccessful_Entrants.docx
they are using:
```
Dear [Title and Surname of account holder]
```
where Title is: Mr or Mrs
Which means that should be: Mr Smith (not full_name)
So next time would be good to use smth like:
```
account_holder = @form_answer.account.owner
@name = "#{account_holder.title} #{account_holder.last_name}"
```

**Comment by rusllonrails** [comment]
@macool looks good :+1: 
Couple of notes above ^ - don't need to fix (I already fixed it)

**Comment by rusllonrails** [comment]
@macool please ignore my comment about account_holder above ^ I'm bit mistaken )

**Pull request merged**